plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/AbstractTextPanelEditor.java

```diff
@@ -191,6 +192,9 @@ public void run() {
             final Action afAction = new AutoFormatAction();
             afAction.setChecked(getPanelSettings().getBoolean(PREF_TEXT_EDITOR_AUTO_FORMAT));
             manager.add(afAction);
+            final Action msAction = new SaveMinifyValue();
```

Comment:
Just realized that we need it only for JSON and XML.
For other formats we mustn't show this option.